packages/patternfly-4/react-core/src/components/Dropdown/DropdownToggle.js

```diff
@@ -7,7 +7,9 @@ import { css } from '@patternfly/react-styles';
 
 const DropdownToggle = ({ children, iconComponent: IconComponent, ...props }) => (
   <Toggle {...props}>
-    {children}
+    <span className={css(styles.dropdownToggleText)} >
```

Comment:
It appears this addition broke the Dropdown - Icon Only case at http://patternfly-react.surge.sh/patternfly-4/components/dropdown.  

![screen shot 2019-03-05 at 2 41 25 pm](https://user-images.githubusercontent.com/895728/53832676-c962aa00-3f54-11e9-8ab3-a7fc0f9c538c.png)

After:
![screen shot 2019-03-05 at 2 41 05 pm](https://user-images.githubusercontent.com/895728/53832685-cd8ec780-3f54-11e9-80dd-ffdcd9d46344.png)

`pf-c-dropdown__toggle-text` should only be added if `children` is text and not an icon.

cc: @spadgett, @janwright73, @priley86

Reply:
yea i just saw this too...